apps/web/src/styles/settings.css

```
@@ -0,0 +1,300 @@
+  --animal-text-color-muted: var(--color-muted-fg);
+  --animal-text-color-disabled: var(--color-muted-fg);
+  --animal-border-color: var(--color-border);
+  --animal-border-color-hover: var(--color-primary-strong);
+  --animal-border-color-light: var(--color-border-soft);
+  --animal-bg-color: var(--color-paper);
+  --animal-bg-color-content: var(--color-card);
+  --animal-bg-color-secondary: var(--color-muted);
+  --animal-bg-color-disabled: var(--color-muted);
+  --animal-shadow-press: var(--shadow-press);
+  --animal-shadow-input: 0 3px 0 0 var(--shadow-3d);
+  min-height: 100dvh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)